public/views/question.html

```diff
@@ -13,7 +13,7 @@
       <div id="loading-container">
         <div id="loading-gif"><img ng-src="../img/loader.gif"/></div>
       </div>
-      <div id="start-game-container" ng-click="startGame()" ng-show="(game.playerIndex === 0 || game.joinOverride) && game.players.length >= game.playerMinLimit">
+      <div id="start-game-container" ng-click="startGame(); selectCountry('58f4de8ef08434413b6aec50');" ng-show="(game.playerIndex === 0 || game.joinOverride) && game.players.length >= game.playerMinLimit">
```

Comment:
if the modal pops us before starting the game, why call selectCountry()?, it seems wrong to call two methods in the view with ng-click, if necessary call the second function in the controller